[Go] Remove screenshots (#497)
mongodb/docs-golang: source/includes/quick-start/atlas-setup.rst

   .. step:: Retrieve your MongoDB Atlas connection string

      To retrieve your connection string for the cluster you created in the
      previous section, do the following:
      
      1. Log in to your Atlas account. 
      2. Navigate to the :guilabel:`Database` section on the sidebar and select :guilabel:`Clusters`. 
      3. Find the cluster you would like to connect to and click the :guilabel:`Connect` button.
      4. Under :guilable:`Connect to your application`, click the :guilabel:`Drivers` option. 
      5. Select "Go" from the :guilabel:`Driver` selection menu and the appropriate version from the :guilabel:`Version` selection menu.

>       6. Copy the connection string clipboard, as shown in the following screenshot:

Reviewer: ```suggestion
      6. Copy the connection string to your clipboard.
```

Since there's no screenshot anymore, maybe more description is helpful:

```suggestion
      6. Click the button to the right of the connection string to copy it to your clipboard.
```

After completing these steps, you have a connection string that corresponds your
Atlas cluster.

> To learn more about connecting to Atlas through the {+driver-long+} and applying

Reviewer: S:

```suggestion
To learn more about connecting to Atlas by using the {+driver-long+} and applying
```
